### PR TITLE
WCAG 2.2 - Fixed reference points draft

### PR DESCRIPTION
**Pull request by alastc**
Initial draft drawn from the [Google doc](https://docs.google.com/document/d/12Zn0_TGcqrM-L_wb0PIFHM4AnHJ64wPsucZyRGlf2Fg/).

Preview:
- [Understanding document](https://raw.githack.com/w3c/wcag/Fixed-Reference-Points/understanding/22/fixed-reference-points.html)
- [HTML Technique](https://raw.githack.com/w3c/wcag/Fixed-Reference-Points/techniques/html/H-Fixed-Reference-Points.html)

[Previous survey](https://www.w3.org/2002/09/wbs/35422/page-break-nav/)
[Last Minutes](https://www.w3.org/2020/03/25-ag-minutes.html#item04)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/pull/1161.html" title="Last updated on Jul 19, 2020, 8:22 PM UTC (99633fa)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wcag/1161/0c37c39...99633fa.html" title="Last updated on Jul 19, 2020, 8:22 PM UTC (99633fa)">Diff</a>

### CODE REVIEW

**Comment by patrickhlauke** [comment]
I'm struggling to see how the "Examples of Fixed Reference Points" in https://raw.githack.com/w3c/wcag/Fixed-Reference-Points/understanding/22/fixed-reference-points.html have anything to do with "a web page or set of web pages is an electronic publication with pagebreak locators"

are they leftovers from some previous, more general, iteration of this SC?

File: understanding/22/fixed-reference-points.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    </section>
+
+
+   <section class="remove">
+   
+      <h2>Fixed Reference Points Normative Text</h2>
+      <blockquote>
+         <p class="conformance-level">A</p>
+
+         <p>When a <a>web page</a> or <a>set of web pages</a> is an <a>electronic publication</a> with <a>pagebreak locators></a>, a mechanism is available to navigate to each locator, and each locator maintains its place in the flow of content, even when the formatting or platform change.</p>
```
Comment (patrickhlauke) [inline understanding/22/fixed-reference-points.html:23]: stray `>` after "pagebreak locators

File: understanding/22/fixed-reference-points.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+         <li>People who are blind or who have low vision or those who may be magnifying content because of a cognitive disability will still be able to find their place and find references to content that are based on page numbers of the print or default version of the publication.</li>
+      </ul>
+      
+   </section>
+   
+   <section id="examples">
+      <h2>Examples of Fixed Reference Points</h2>
+
+      <ul>
+         <li>A sortable list of elements may, after focussing a list element, provide adjacent controls for moving the element up or down in the list by simply tapping/clicking on those controls.</li>
```
Comment (patrickhlauke) [inline understanding/22/fixed-reference-points.html:72]: as noted, these examples don't seem to have anything to do with "a web page or set of web pages is an electronic publication with pagebreak locators"

**Comment by detlevhfischer** [comment]
There are examples for SC Dragging included at the bottom of this PR...

**Comment by alastc** [comment]
Ach, sorry. The google doc didn't have any examples and I missed that as part of moving it into github.

I've quickly done a very basic example we can build on later.

**Pull request merged**